gems/pending/spec/appliance_console/database_replication_primary_spec.rb

```diff
@@ -0,0 +1,112 @@
+require 'appliance_console/logging'
+require 'appliance_console/prompts'
+require 'appliance_console/database_configuration'
+require 'appliance_console/database_replication'
+require 'appliance_console/database_replication_primary'
+require 'linux_admin'
+require 'pg'
+
+NETWORK_INTERFACE = "eth0".freeze
```

Comment:
I don't think both this and [this](https://github.com/ManageIQ/manageiq/pull/9585/files#diff-3f47ed162edfdbfb5f8739969fbd03f8R12) should be necessary.